src/charts/vertical_bar.coffee

```diff
@@ -191,6 +191,14 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
     _.uniq _.flatten(groups)
   .property 'groupedData.@each'
 
+  labelIDMapping: Ember.computed ->
+    @get('individualBarLabels').map (label, i) ->
+      labelLookup = {}
+      labelLookup[label] = i
+      labelLookup
+    .reduce(Ember.merge)
+  .property 'individualBarLabels'
```

Comment:
:+1: `individualBarLabels` is made by `map` so it's reference would update whenever it changes but you're right it should be `individualBarLabels.[]`